etna/core/models/basepage.py

```diff
@@ -158,12 +158,17 @@ def mourning_notice(self):
 
         return MourningNotice.objects.first()
 
+    @cached_property
+    def type(self):
+        return self.__class__.__name__
```

Comment:
This only gives a value like `ArticlePage` rather than the more "usual" `articles.ArticlePage`. Should we make this consistent?